README.md

```diff
@@ -133,8 +136,11 @@ Options:
                                   preset: A5, A4, A3, B5, B4, JIS-B5, JIS-B4, letter, legal,
                                   ledger
                                   custom(comma separated): 182mm,257mm or 8.5in,11in
-  --style <stylesheet>            additional stylesheet URL or path
-  --user-style <user_stylesheet>  user stylesheet URL or path
+  --crop-marks                    print crop marks
+  --bleed <bleed>                 bleed [3mm]
+  --crop-offset <offset>          crop offset [auto]
+  --style <stylesheet>            additional stylesheet URL or path or CSS code
+  --user-style <user_stylesheet>  user stylesheet URL or path or CSS code
```

Comment:
Personally, I don't feel natural setting with the same option name to specify style files by URL/file and directly by CSS. The implementation that switching its behavior by `isCssString` is too naive so it may confuse users when an unintentional stylesheet value is set.

One of my ideas is to provide the new option something likes `--css <code>`, which simply set `viewerCustomStyle` that is placed after `/*</viewer>*/`. How do you think?